libs/full/parcelport_lci/include/hpx/parcelport_lci/receiver.hpp

```diff
@@ -17,15 +17,57 @@
 #include <hpx/parcelport_lci/receiver_connection.hpp>
 
 #include <algorithm>
+#include <cstddef>
+#include <cstring>
 #include <deque>
 #include <iterator>
 #include <list>
 #include <memory>
 #include <mutex>
 #include <set>
 #include <utility>
+#include <vector>
 
 namespace hpx::parcelset::policies::lci {
+    struct BufferWrapper
```

Comment:
Could you please conform to the HPX naming conventions (not `CamelCase`, but `snake_case`) (see: https://github.com/STEllAR-GROUP/hpx/blob/master/.github/CONTRIBUTING.md#general-info)?